README.md

```diff
@@ -35,10 +35,18 @@ install(RabbitMQ) {
       }
     }
     
-    //consume example
+    //consume with autoack example
     rabbitConsumer {
-      consume<MyObject>("queue") { consumerTag, body ->
-         println("Consumed message $body, consumer tag: $consumerTag")
+      consume<MyObject>("queue") { consumerTag, body, _channel, _envelope ->
```

Comment:
@JUtupe Does this syntax even work? In tests I see this:
```
            rabbitConsumer {
                consume("queue", true, consumer)
            }
```

Shouldn't it be the one we document?

Reply:
`consumer` is a simple lambda. Both 
```
rabbitConsumer {
    consume("queue", true, consumer)
}
```
and
rabbitConsumer {
    consume("queue", true) { tag, body ->
        ...
    }
}
should work properly.